documentation/SyntaxChangelog.md

```diff
@@ -0,0 +1,118 @@
+     * Added parameter `-DisplayName`
+  * Modified cmdlet `New-AzDevCenterAdminEnvironmentType`:
+     * Added parameter `-DisplayName`
+  * Modified cmdlet `New-AzDevCenterAdminPool`:
+     * Added parameters `-DisplayName`, `-ManagedVirtualNetworkRegion`, `-SingleSignOnStatus`, `-VirtualNetworkType`
+  * Modified cmdlet `New-AzDevCenterAdminProject`:
+     * Added parameter `-DisplayName`
+  * Modified cmdlet `New-AzDevCenterUserDevBox`:
+     * Removed parameter `-DevCenter`
+     * Added parameter `-DevCenterName`
+  * Modified cmdlet `New-AzDevCenterUserEnvironment`:
+     * Removed parameter `-DevCenter`
+     * Added parameters `-DevCenterName`, `-ExpirationDate`
+  * Modified cmdlet `Remove-AzDevCenterUserDevBox`:
+     * Removed parameter `-DevCenter`
+     * Added parameter `-DevCenterName`
+  * Modified cmdlet `Remove-AzDevCenterUserEnvironment`:
+     * Removed parameter `-DevCenter`
+     * Added parameter `-DevCenterName`
+  * Modified cmdlet `Restart-AzDevCenterUserDevBox`:
+     * Removed parameter `-DevCenter`
+     * Added parameter `-DevCenterName`
+  * Modified cmdlet `Skip-AzDevCenterUserDevBoxAction`:
+     * Removed parameters `-DevCenter`, `-ActionName`
+     * Added parameters `-DevCenterName`, `-Name`
+  * Modified cmdlet `Start-AzDevCenterUserDevBox`:
+     * Removed parameter `-DevCenter`
+     * Added parameter `-DevCenterName`
+  * Modified cmdlet `Stop-AzDevCenterUserDevBox`:
+     * Removed parameter `-DevCenter`
+     * Added parameter `-DevCenterName`
+  * Modified cmdlet `Update-AzDevCenterAdminCatalog`:
+     * Added parameter `-SyncType`
+  * Modified cmdlet `Update-AzDevCenterAdminDevCenter`:
+     * Added parameter `-DisplayName`
+  * Modified cmdlet `Update-AzDevCenterAdminEnvironmentType`:
+     * Added parameter `-DisplayName`
+  * Modified cmdlet `Update-AzDevCenterAdminPool`:
+     * Added parameters `-DisplayName`, `-ManagedVirtualNetworkRegion`, `-SingleSignOnStatus`, `-VirtualNetworkType`
+  * Modified cmdlet `Update-AzDevCenterAdminProject`:
+     * Added parameter `-DisplayName`
+  * Added cmdlet `Get-AzDevCenterAdminCatalogSyncErrorDetail`.
+  * Added cmdlet `Get-AzDevCenterAdminEnvironmentDefinition`.
+  * Added cmdlet `Get-AzDevCenterAdminEnvironmentDefinitionErrorDetail`.
+  * Added cmdlet `Get-AzDevCenterUserDevBoxOperation`.
+  * Added cmdlet `Get-AzDevCenterUserEnvironmentAction`.
+  * Added cmdlet `Get-AzDevCenterUserEnvironmentLog`.
+  * Added cmdlet `Get-AzDevCenterUserEnvironmentOperation`.
+  * Added cmdlet `Get-AzDevCenterUserEnvironmentOutput`.
+  * Added cmdlet `Invoke-AzDevCenterUserDelayEnvironmentAction`.
+  * Added cmdlet `Repair-AzDevCenterUserDevBox`.
+  * Added cmdlet `Skip-AzDevCenterUserEnvironmentAction`.
+  * Added cmdlet `Start-AzDevCenterAdminPoolHealthCheck`.
+  * Added cmdlet `Update-AzDevCenterUserEnvironment`.
+#### Az.HDInsight 6.1.0 
+* Modified cmdlet `New-AzHDInsightCluster`:
+     * Added parameter `-EnableSecureChannel`
```

Comment:
One slight inconsistency: line 102 ends with ".", line 105 doesn't.

Reply:
Should we consider removing all the ending ":" and "."?

Reply:
I have made the adjustments; you can come and see the effects.